Seaglass/Controller/Main View/MainViewRoomsController.swift

```diff
@@ -137,28 +137,43 @@ class MainViewRoomsController: NSViewController, MatrixRoomsDelegate, NSTableVie
         } else {
             cell?.RoomListEntryIcon.setAvatar(forRoomId: state.roomId)
         }
-        
+
         var unreadColor = NSColor(calibratedRed: 0.51, green: 0.61, blue: 0.95, alpha: 1.00)
         if state.isInvite() {
             unreadColor = NSColor(calibratedRed: 0.90, green: 0.35, blue: 0.29, alpha: 1.00)
             cell?.RoomListEntryTopic.stringValue = "Room invite"
             cell?.RoomListEntryUnread.isHidden = false
         } else {
-            var memberString: String = ""
-            var topicString: String = "No topic set"
+            var topicMode = "lastMessagePreview" // TODO: set this via user preferences
```

Comment:
for now let's use let since it isn't configurable 
```suggestion
            let topicMode = "lastMessagePreview" // TODO: set this via user preferences
```

Reply:
actually never mind we should probably use a NSUserDefault to set the mode. If you don't know how to do that its fine, I'll make a PR afterwards.

If you do want to try, see https://developer.apple.com/documentation/foundation/userdefaults or https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/UserDefaults/AccessingPreferenceValues/AccessingPreferenceValues.html#//apple_ref/doc/uid/10000059i-CH3-SW1

Reply:
Yeah, not sure how to approach that given my limited swift foo.